nncf/common/compression.py

```diff
@@ -282,3 +284,27 @@ def _get_state_without_name(self) -> Dict[str, Any]:
 
     def _parse_bn_adapt_params(self) -> Optional[Dict]:
         return extract_bn_adaptation_init_params(self.config, self.name)
+
+    def _check_scopes_in_graph(self, graph: NNCFGraph) -> None:
```

Comment:
Why put this function into a class? This limits the usage of the function only within the builder-derived classes, and since it's a protected function in a base class, it cannot be used in a stand-alone fashion. If it should belong to a class, then putting it into `NNCFGraph` seems to be a better decision. I'm also OK with keeping it a stand-alone fn for the time being.

Reply:
Move to `common.scopes` as function.